src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/wizard.ts

```diff
@@ -45,35 +43,6 @@ export const useWizardStore = defineStore("wizard", {
     currentStepStage(state): Components.Schemas.PortalStage | ReferenceStage {
       return this.steps[state.step - 1].stage;
     },
-    validationState(state): PortalStageValidation {
```

Comment:
Do we need `hasDuplicateReferences` in application store still?

Reply:
Ahh good catch, we don't.

Reply:
removed.